mopidy/mpd/protocol/playback.py

```diff
@@ -46,11 +45,10 @@ def mixrampdb(context, decibels):
     volume so use negative values, I prefer -17dB. In the absence of mixramp
     tags crossfading will be used. See http://sourceforge.net/projects/mixramp
     """
-    pass
+    raise exceptions.MpdNotImplemented  # TODO
```

Comment:
Same goes for these, that is I would have preferred that these be a separate commit with just "Make mixramp commands raise not implemented" instead of squashing everything into a single commit. And just to be clear I don't expect you to fix this for this PR/commit, just keep it in mind next time around :-)